skypack/README.md

```diff
@@ -0,0 +1,7 @@
+# Skypack
+
+This is a work-in-progress SDK for interacting with the Skypack Package CDN. More documentation will be written as this gets closer to a public release!
+
+```
+npm install skypack
+```
```

Comment:
Maybe at least add a usage section? Even if the SDK is still early in development, we could have a discussion about the APIs, the earlier the better in my experience